.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,36 @@
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+| Name| about |title|labels|
+| ------------- | ------------- |------------- |------------- |
+| Feature Request  | Suggest an idea for this project  |[Short Description] (Version: [AYS]) |enhancement|
+
+**Read before creating an issue**
+In general, we close any issues that are
+* unactionable(fill the template below under "Description")
+* questions best served elsewhere
+  We have a small community of individuals responsible for managing issues, and our goal is to ensure that the open issues maintain a high level of quality.
+
+* **Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
+
+# A Description
+## Explain in detail the additional functionality you would like to see in AYS
+Be descriptive, including the interface you'd like to see, as well as any suggestions you may have with regard to its implementation.
+
+## Explain why this feature is important
+If it's related to some problem you're having, give a clear and concise description of what the problem is.
+
+## Describe the solution you'd like
+A clear and concise description of what you want to happen.
+
+##  Describe alternatives you've considered
+A clear and concise description of any alternative solutions or features you've considered.
+
+## Additional context
+Add any other context or screenshots about the feature request here.
```

Comment:
Can you revise this explanation as shown below?

`Add any other context about the feature request here with supporting any gist or attachment that would make it easier to understand the feature which you mention`

Reply:
I revised this part.